static/locales/en_GB.yaml

```diff
@@ -649,15 +647,12 @@ Video:
   Copy YouTube Embedded Player Link: 'Copy YouTube Embedded Player Link'
   Open in Invidious: 'Open in Invidious'
   Copy Invidious Link: 'Copy Invidious Link'
-  View: 'View'
   Views: 'Views'
   Loop Playlist: 'Loop Playlist'
   Shuffle Playlist: 'Shuffle Playlist'
   Reverse Playlist: 'Reverse Playlist'
   Play Next Video: 'Play Next Video'
   Play Previous Video: 'Play Previous Video'
-  # Context is "X People Watching"
```

Comment:
I do see that we're not including this comment anymore, which I think is fine as the pluralization and placement makes the context of the translation label more intuitive. As a sidenote, comments like these do make me wonder if having implicit subjects/objects in our labels is a burden for [a subset of] our translators; I have no idea.